terraform/deployments/cluster-infrastructure/vpc.tf

```diff
@@ -99,14 +99,8 @@ resource "aws_eip" "eks_nat" {
   # TODO: depends_on = [aws_internet_gateway.gw] once we've imported the IGW from govuk-aws.
 }
 
-import {
-  for_each = var.eks_licensify_gateways
-  to       = aws_eip.eks_nat[each.key]
-  id       = each.value.eip
-}
-
 resource "aws_nat_gateway" "eks" {
```

Comment:
I'm pretty sure the resource name will need updating to `eks_licensify` for the moved block below to work

Reply:
Ah yeah, I fluffed that names.. Should be fixed now and the terraform plan looks like what we want :)